test/rubygems/test_gem_commands_uninstall_command.rb

```diff
@@ -261,6 +291,25 @@ def test_handle_options_vendor
     assert_match expected, @ui.error
   end
 
+  def test_execute_two_version
+    @cmd.options[:args] = %w[a b]
+    @cmd.options[:version] = Gem::Requirement.new("> 1")
+
+    use_ui @ui do
+      e = assert_raises Gem::MockGemUi::TermError do
+        @cmd.execute
+      end
+
+      assert_equal 1, e.exit_code
+    end
+
+    msg = "ERROR:  Can't use --version with multiple gems. You can specify multiple gems with" \
+      " version requirments using `gem install 'my_gem:1.0.0' 'my_other_gem:~>2.0.0'`"
```

Comment:
"requirements"

Reply:
@bronzdoc

Sorry, I was in the test file and missed that.  Two items:

1. @deivid-rodriguez just copied it from here, where the typo also exists:

https://github.com/rubygems/rubygems/blob/4dc3d9d5d1a92f5b1b056716900057b016d9ea1b/lib/rubygems/commands/install_command.rb#L144-L145

2. In the two uninstall files here, should "gem install' be changed to "gem uninstall" in the 'messages'?

Reply:
@bronzdoc

`install_command.rb` is modified in #2452, should I fix the typo there?